packages/x-date-pickers/src/DesktopNextDatePicker/DesktopNextDatePicker.tsx

```diff
@@ -45,6 +45,7 @@ const DesktopNextDatePicker = React.forwardRef(function DesktopNextDatePicker<TD
     viewRenderers,
     format: getDatePickerFieldFormat(utils, defaultizedProps),
     showToolbar: defaultizedProps.showToolbar ?? false,
+    yearsPerRow: 4,
```

Comment:
`yearPerRow` still does not seem to like being overridden on new picker component. 🤷 
![Screenshot 2023-01-11 at 13 32 01](https://user-images.githubusercontent.com/4941090/211797755-52584122-f1a3-4594-aac7-d44f51223f0d.png)
https://codesandbox.io/s/sweet-hellman-c6lj5k?file=/demo.tsx

Reply:
:facepalm: sorry I fixed it to quickly
Fixed

Reply:
Does still seem to be an issue with `StaticNextDatePicker` and `StaticNextDateTimePicker`. 🙈